### ISSUE TITLE
Anywidget for JavaScript kernel

### ISSUE THREAD

**Issue by MRYingLEE** [manzt/anywidget#450]
Now there is a JavaScript kernel for Jupyterlite, https://github.com/jupyter-xeus/xeus-javascript.

For Anywidget is developed in TypeScript, maybe it is easy to be adapted to run in the JavaScript kernel in an even simpler way.

Of course, there should be some backend work. If no one works on it. I will do it.

Thanks,

**Comment by manzt** [comment]
Want to have a look at the backend work for the Deno kernel? I bet there is a lot that can be reused if you'd like to try creating such an integration:

https://github.com/manzt/anywidget/tree/main/packages/deno

**Comment by MRYingLEE** [comment]
It is great that AnyWidget can run on Deno now. 

I read all contents on https://anywidget.dev/, but I didn't expect you had worked on JavaScript kernel already. It is worth mentioning in the Readme.md.

Thanks.

**Comment by manzt** [comment]
Good point! I just haven't advertised widely as it is very experimental. I'll get on adding it to the readme.

**Comment by MRYingLEE** [comment]
FYI. xeus-javascript is working to support Anywidget (https://github.com/jupyter-xeus/xeus-javascript/issues/17).
